data/story northern union.txt

```diff
@@ -198,7 +201,7 @@ mission "Meeting an Alpha 1"
 			`	You cut the commlink, relieved that it was that easy.`
 	
 	on complete
-		dialog `Soon after you land, a courier comes to your ship and pays you <payment> in exchange for your sensor logs regarding the <npc>. She says, "If you would be interested in continuing to work for my Godlord, render yourself before him at the Rusted Wheel Cafe. He has a gift for you."`
+		dialog `Soon after you land, a courier comes to your ship and pays you <payment> in exchange for your sensor logs regarding the <npc>. "If you would be interested in continuing to work for my Godlord, render yourself before him at the Rusted Wheel Cafe. He has a gift for you."`

```

Comment:
I think it's better to revert this change. Having a quote from a courier on its own without using either '[person] says, "Quote."' or '"Quote," [person] says.' reads a bit awkward, and I'm not sure if this is even correct grammar.